trackintel/io/postgis.py

```diff
@@ -139,55 +167,59 @@ def write_triplegs_postgis(
         How many entries should be written at the same time.
 
     if_exists : str, {'fail', 'replace', 'append'}, default 'fail'
-        What should happen if the table already exists.
+        How to behave if the table already exists.
+
+        - fail: Raise a ValueError.
+        - replace: Drop the table before inserting new values.
+        - append: Insert new values to the existing table.
 
     Examples
     --------
-    >>> df.as_triplegs.to_postgis(conn_string, table_name)
+    >>> tpls.as_triplegs.to_postgis(conn_string, table_name)
+    >>> ti.io.postgis.write_triplegs_postgis(tpls, conn_string, table_name)
     """
-    engine = create_engine(conn_string)
-    conn = engine.connect()
-    try:
-        triplegs.to_postgis(table_name, conn, if_exists=if_exists, index=True, chunksize=sql_chunksize)
-    finally:
-        conn.close()
+    triplegs.to_postgis(table_name, con, if_exists=if_exists, schema=schema, index=True, chunksize=sql_chunksize)
```

Comment:
Maybe we can add the missing arguments from the original geopandas [function](https://geopandas.org/docs/reference/api/geopandas.GeoDataFrame.to_postgis.html) as optional input just in case. 
This would concern these two arguments: 
index_label=None, dtype=None

Our default name for the index is the index name (which I think is very good)